Source/WebCore/xml/XPathEvaluator.h

```diff
@@ -41,6 +41,7 @@ class XPathEvaluator : public RefCounted<XPathEvaluator> {
 
     ExceptionOr<Ref<XPathExpression>> createExpression(const String& expression, RefPtr<XPathNSResolver>&&);
     Ref<XPathNSResolver> createNSResolver(Node& nodeResolver);
+    void createNSResolverForBindings(Node&) const { } // Legacy.
```

Comment:
I might suggest `static` instead of `const` since this is just to compile nothing.

Reply:
Will fix, thanks.